src/main/java/net/datafaker/providers/base/Finance.java

```diff
@@ -1,11 +1,9 @@
 package net.datafaker.providers.base;
 
+import net.datafaker.annotations.Deterministic;
+
 import java.math.BigInteger;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
+import java.util.*;
```

Comment:
Please leave the specific imports vs. Wildcard

Reply:
Done

Reply:
I don't think that's a styling rule we have. We've got multiple `*` imports in our code.

Reply:
I had to update my IDE settings, also prefer specific imports so there's that

Reply:
but thanks for addressing it anyway. I'm going to merge this, if you want you can try the snapshot version in a few minutes, and if all is well, happy to create a release out of this.